firebase-firestore/CHANGELOG.md

```diff
@@ -1,19 +1,28 @@
 # Unreleased
-- [feature] Added `FieldValue.increment()`, which can be used in `update()`
-  and `set(..., SetOptions.merge())` to increment or decrement numeric field
-  values safely without transactions.
-- [fixed] Fixed calculation of SQLite database size on Android 9 Pie devices.
-  Previous method could be off by a few MBs on these devices, potentially
-  delaying garbage collection.
+
+# 18.2.0
+- [feature] Added `FieldValue.increment()`, which can be used in `update()` and
+  `set(..., SetOptions.merge())` to increment or decrement numeric field values
+  safely without transactions.
+- [feature] Added functional interface `FirebaseFirestore.runBatch()`, similar
+  to `FirebaseFirestore.runTransaction()`, which allows a developer to focus on
+  the mutations of the batch rather than on creating and committing the batch.
 - [changed] Prepared the persistence layer to support collection group queries.
-  While this feature is not yet available, all schema changes are included
-  in this release.
+  While this feature is not yet available, all schema changes are included in
+  this release.
 - [changed] Added `@RestrictTo` annotations to discourage the use of APIs that
-  are not public. This affects internal APIs that were previously obfuscated
-  and are not mentioned in our documentation.
+  are not public. This affects internal APIs that were previously obfuscated and
+  are not mentioned in our documentation.
 - [changed] Improved error messages for certain Number types that are not
   supported by our serialization layer (#272).
 
+# 18.1.0
+- [changed] Internal changes to ensure functionality alignment with other SDK
+  releases.
```

Comment:
I thought we only used this message when we had no "real" changes to list, but apparently (below) we do in this case...?

I don't care strongly either way though.

Reply:
It's in the public release notes, so might as well leave it. https://firebase.google.com/support/release-notes/android

Reply:
Yeah, I asked about that too. The decision at the time was to put it in all sdks, even ones like ours that didn't *really* need it.